boards/arm/mimxrt1064_evk/board.cmake

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+board_runner_args(jlink "--device=MCIMXRT1064")
```

Comment:
Does jlink support this device yet? I don't see it in https://www.segger.com/downloads/supported-devices.php

Reply:
Segger claimed to support RT1064 in jlink v6.40, however the flash loader only supports the external flash (on board). I filed a ticket requesting the support of programming the internal flash, and they'll be working on it. Anyway, the device name in this arg is not correct, I'll update it.

Reply:
How do you program the application to internal flash? The device name in `board.cmake` gets passed to the JLink Commander when you run `ninja flash` or `ninja debug`, so it needs to match one of the device names listed on the segger website.

If we support ITCM only for now, then you can change this to:
```
board_runner_args(jlink "--device=Cortex-M7")
```
like we had in rt1050 prior to commit 7433f75e1a87a194208b1659430a8e124dee9cb2

Reply:
I was using a flash loader I created for segger. I'll change it to cortex-m7.